### DISCUSSION TITLE
弹幕功能优化

### DISCUSSION THREAD

**Discussion by loikuan** [ywmoyue/biliuwp-lite#108] [feature]
像手机客户端一样可以过滤不同等级用户的弹幕

可以自定义弹幕的字体

**Comment by ywmoyue** [comment]
弹幕库不在这个仓库中维护，可以到这里提issue： https://github.com/xiaoyaocz/NSDanmaku

**Comment by ywmoyue** [comment]
之前没注意这块实现，播放器中是有弹幕屏蔽等级和缩放设置的 @loikuan 
![image](https://github.com/ywmoyue/biliuwp-lite/assets/28388300/fe471889-1804-4b58-9f28-cab117b1d86f)
